common/culture/innovations/wc_cultural_maa_innovations.txt

```diff
@@ -622,6 +603,8 @@ innovation_farstriders = {
 	}
 
 	unlock_maa = farstriders
+	unlock_maa = dragonhawk_rider
+	unlock_maa = spellbreaker
```

Comment:
I thought, in a previous discussion we said that spellbreakers are only a thing after the the fall of silvermoon. so we shouldn't put them in the same innovation...

Reply:
> I thought, in a previous discussion we said that spellbreakers are only a thing after the the fall of silvermoon. so we shouldn't put them in the same innovation...

Yep.

Reply:
What date should elves get spellbreakers?

Reply:
> @zumbak04 @arithon What date should elves get spellbreakers?

After the Fall of Quel'thalas.